DataFormats/GEMRecHit/interface/ME0Segment.h

```diff
@@ -72,7 +73,10 @@ class ME0Segment final : public RecSegment {
 
   float time() const    { return theTimeValue; }
   float timeErr() const { return theTimeUncrt; }
-    
+
+  float deltaPhi() const { return theDeltaPhi; }
+  static float computeDeltaPhi(const ME0Chamber * chamber, const LocalPoint& position, const LocalVector& direction );
```

Comment:
I can put it in the ME0SegmentAlgorithm, if there is no problem that RecoMuonProducer depends on it. Otherwise I can duplicate the code and place it both in the global muon reconstruction and in the segment algorithm.